source/wp-content/themes/wporg-news-2021/block-template-parts/content-category-month-in-wordpress.html

```diff
@@ -0,0 +1,11 @@
+<!-- wp:group {"tagName":"header","className":"entry-header"} -->
+<header class="wp-block-group entry-header">
+	<h2><!-- wp:post-date {"format":"M"} /--></h2>
+
+	<!-- wp:group {"className":"entry-meta"} -->
+	<div class="wp-block-group entry-meta">
+		<!-- wp:post-date {"format":"Y"} /-->
```

Comment:
This'll need to be a link, but I don't think there's a block that would create one w/ the post date as anchor text, so you might have to use the HTML block and do it manually

Reply:
After much searching, it turns out that `<!-- wp:post-date {"isLink":true} /-->` does the job.